examples/hello_world.rs

```diff
@@ -1,12 +1,12 @@
 type Exception = Box<dyn std::error::Error + Send + Sync + 'static>;
+use async_std::task;
 
-#[runtime::main]
-async fn main() -> Result<(), Exception> {
-    femme::start(log::LevelFilter::Info)?;
+fn main()  {
```

Comment:
This should probably return:
```suggestion
fn main() -> Result<(), surf::Exception> {
```

so that `unwrap` isn't needed.